timemachine/fe/free_energy.py

```diff
@@ -379,9 +380,6 @@ def estimate_free_energy_pair_bar(
         df, df_err = bar_with_bootstrapped_uncertainty(w_fwd, w_rev)  # reduced units
         dG, dG_err = df / beta, df_err / beta  # kJ/mol
 
-        message = f"{prefix} BAR: lambda {lamb_idx} -> {lamb_idx + 1} dG: {dG:.3f} +- {dG_err:.3f} kJ/mol"
-        print(message, flush=True)
```

Comment:
Thinking to remove this to simplify the code; it doesn't seem very useful in the current state, since diagnostics are only printed in the final analysis loop after the simulations have run (for both sequential and bisection methods).